src/amo/pages/UserProfileEdit/index.js

```diff
@@ -653,7 +653,7 @@ export class UserProfileEditBase extends React.Component<Props, State> {
                     )
                   : i18n.sprintf(
                       i18n.gettext(`Introduce %(username)s to the community`),
-                      { username },
+                      { username: user ? user.username : userId },
```

Comment:
Same comment about `userId` not being particularly useful here.

Reply:
I changed to use a `LoadingText` component when there is no user:

![screen shot 2018-11-19 at 12 15 26](https://user-images.githubusercontent.com/217628/48703714-e6b57780-ebf4-11e8-8624-dfbd4030609a.png)